docs/benchmarks/profiling.md

```diff
@@ -10,7 +10,7 @@ contributors to the `hydra-node`.
 On every PR and also for the latest `master`, we do compute typical transaction
 costs in size, memory and cpu usage of the Hydra protocol transactions on
 Cardano. The latest results can be seen
-[here](https://hydra.family/head-protocol/benchmarks/transaction-cost/).
+[here](/benchmarks/transaction-cost/).
```

Comment:
Is there a reason to change to absolute url? For me both are fine just curious.

Reply:
It is a good question, and [here](https://docusaurus.io/docs/versioning#link-docs-by-file-paths) is the reason behind it.